tests/testthat/test-reactivity.r

```diff
@@ -126,6 +126,23 @@ test_that("ReactiveValues", {
   expect_error(values$a <- 1)
 })
 
+test_that("reactiveValues keys are sorted", {
+  values <- reactiveValues(b=2, a=0)
+  values$C <- 13
+  values$A <- 0
+  values$c <- 3
+  values$B <- 12
+  # Setting an existing value shouldn't change order
+  values$a <- 1
+  values$A <- 11
```

Comment:
I guess we don't have the notion of removing a value? That's the only other case I could think of.

Reply:
Right, we don't have a way to remove a value.